src/styles/blocks/card.scss

```diff
@@ -0,0 +1,135 @@
+.card {
```

Comment:
i'm bit concerned with overall card construct
would it be much easier if your entire card has `padding: 32px 16px 16px`
so all the children are already limited and it simplifies a lot of styling